packages/rsk3-utils/src/index.js

```diff
@@ -1046,6 +1050,81 @@ const soliditySha3 = function() {
     return keccak256(`0x${hexArguments.join('')}`);
 };
 
+/**
+ * Gets the RSK address from BTC private key
+ *
+ * @method getRskAddress
+ *
+ * @param {String} btcPrivateKey BTC Private key.
+
+ * @return {String} RSK address.
+ */
+const getRskAddress = function(btcPrivateKey) {
+    const myWallet = wallet.fromPrivateKey(Buffer.from(_keyBtcToRskInBytes(btcPrivateKey)));
+    const addressInRskFormat = myWallet.getAddress();
+    return addressInRskFormat.toString('hex');
+};
+
+/**
+ * Get RSK Private Key from BTC Private Key
+ *
+ * @param {String} btcPrivateKey BTC Private key.
+ *
+ * @return {String} RSK Private Key.
+ */
+function getRskPrivateKey(btcPrivateKey) {
+    var privKeyBytes = _keyBtcToRskInBytes(btcPrivateKey);
+    var privKeyInRskFormat = Buffer.from(privKeyBytes).toString('hex');
+    return privKeyInRskFormat;
+}
+
+const _keyBtcToRskInBytes = function(privKeyAsExportedByBitcoinDumpprivkey) {
+    var decodedKey = bs58.decode(privKeyAsExportedByBitcoinDumpprivkey);
+    var privKeyBytes = decodedKey.slice(1, decodedKey.length - 5);
+    return privKeyBytes;
+};
+
+/**
+ * Gets BTC private key from RSK private key.
+ *
+ * @method getBtcPrivateKey
+ *
+ * @param {bool} isBtcMainnet Respresents BTC mainnet network.
+ *
+ * @param {String} rskPrivateKey RSK Private Key.
+ *
+ * @return {String} BTC address.
+ */
+
+const getBtcPrivateKey = function(isBtcMainnet, rskPrivateKey) {
+    var addressArray = convertHex.hexToBytes(rskPrivateKey);
```

Comment:
before performing the conversion, validate that the input value, `rskPrivateKey`.